extensions/ql-vscode/src/vscode-tests/no-workspace/query-results.test.ts

```diff
@@ -18,7 +18,7 @@ import { slurpQueryHistory, splatQueryHistory } from '../../query-serialization'
 chai.use(chaiAsPromised);
 const expect = chai.expect;
 
-describe('query-results', () => {
+describe.only('query-results', () => {
```

Comment:
```suggestion
describe('query-results', () => {
```

Reply:
Thanks for catching that and re-reviewing.